angular-client/src/pages/map/map.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+import { MapService } from './map.service';
+
+@Component({
+  selector: 'app-map',
```

Comment:
why app?

Reply:
that was the name given in the tutorial, its just what the component is going to be addressed by, changed it